packages/plugins/rubygem-recursive-open-struct/rubygem-recursive-open-struct.spec

```
@@ -1,70 +1,53 @@
-# Generated from recursive-open-struct-1.1.0.gem by gem2rpm -*- rpm-spec -*-
-# template: scl
-%{?scl:%scl_package rubygem-%{gem_name}}
-%{!?scl:%global pkg_name %{name}}
-
+# template: default
 %global gem_name recursive-open-struct
 
-Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 1.1.0
-Release: 2%{?dist}
+Name: rubygem-%{gem_name}
+Version: 1.1.3
+Release: 1%{?dist}
 Summary: OpenStruct subclass that returns nested hash attributes as RecursiveOpenStructs
-Group: Development/Languages
 License: MIT
 URL: https://github.com/aetherknight/recursive-open-struct
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 
 # start specfile generated dependencies
-Requires: %{?scl_prefix_ruby}ruby(release)
-Requires: %{?scl_prefix_ruby}ruby
-Requires: %{?scl_prefix_ruby}ruby(rubygems)
-BuildRequires: %{?scl_prefix_ruby}ruby(release)
-BuildRequires: %{?scl_prefix_ruby}ruby
-BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+Requires: ruby
+BuildRequires: ruby
+BuildRequires: rubygems-devel
 BuildArch: noarch
-Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
 # end specfile generated dependencies
 
 %description
 RecursiveOpenStruct is a subclass of OpenStruct. It differs from
 OpenStruct in that it allows nested hashes to be treated in a recursive
-fashion.
+fashion. For example:
+ros = RecursiveOpenStruct.new({ :a => { :b => 'c' } })
+ros.a.b # 'c'
+Also, nested hashes can still be accessed as hashes:
+ros.a_as_a_hash # { :b => 'c' }.
```

I'd drop the example
```suggestion
fashion.
```